### ISSUE TITLE
Question about loss function in D3PM

### ISSUE THREAD

**Issue by loongzeng** [cloneofsimo/d3pm#1]
Hello,

I hope this message finds you well. I have a question regarding the implementation of the loss function in D3PM. Specifically, in the forward method, there is a line of code located [here](https://github.com/cloneofsimo/d3pm/blob/421a3e972a8e33a79fc188c8b91b85a03c1fe274/d3pm_runner.py#L268):

```
predicted_x0_logits = self.model_predict(x, t, cond)
```

I was wondering why it isn't `self.model_predict(x_t, t, cond)` instead, predicting x_0 through x_t?

Thank you for your attention to this matter.

**Comment by cloneofsimo** [comment]
Awesome, you are right on this issue.

**Comment by cloneofsimo** [comment]
Will add a pr on this

**Comment by hummat** [comment]
If I'm not mistaken, this is not fixed yet, right?
